packages/app/src/pages/api/notion.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import { NotionAPI } from 'notion-client';
+const api = new NotionAPI();
```

Comment:
we use `next export` to export static files, so api feature not work in our project
is recommanded implement this api in [OctoBase](https://github.com/toeverything/octobase)

Reply:
Ah, OK with me. Maybe I should close this pr for now and try octobase way. And consider together with client-side development. 

BTW, notion-client is very useful for any public notion page. I think it's better to implement that in Octobase. cc @joebeijing